### PR TITLE
KOB-10596 [Automation] Create Sample AutomationName Flutter

### PR DESCRIPTION
**Pull request by hyndole**
Sample for AutomationName Flutter

### CODE REVIEW

File: flutter/javascript/.babelrc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "presets": [ "es2015", "stage-0" ],
+    "plugins": [
+        ["transform-runtime", {
+          "regenerator": true
+        }]
+      ]
+  }
```
Comment (nghiadhd) [inline flutter/javascript/.babelrc]: Add newline

File: flutter/javascript/package.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "appium",
```
Comment (nghiadhd) [inline flutter/javascript/package.json:2]:
```suggestion
  "name": "appium-flutter-driver-sample",
```

File: flutter/javascript/src/index.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        app: 'https://kobiton-devvn.s3-ap-southeast-1.amazonaws.com/apps-test/demo/Runner.zip',
+        newCommandTimeout: 360,
+        waitForQuiescence: false,
+        autoAcceptAlerts: true
+      }
+    : {};
+
+const opts = {
+  protocol: 'https',
+  hostname: 'username:apiKey@api.kobiton.com',
```
Comment (nghiadhd) [inline flutter/javascript/src/index.js:33]: Define const for username & token

`const username = process.env.KOBITON_USERNAME || ''
const apiKey = process.env.KOBITON_API_KEY || ''
const deviceName = process.env.KOBITON_DEVICE_NAME || ''

const kobitonServerConfig = {
  protocol: 'https',
  host: 'api.kobiton.com',
  auth: `${username}:${apiKey}`
}`

File: flutter/javascript/src/index.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  await driver.touchAction({
+    action: 'tap',
+    element: { elementId: buttonFinder }
+  });
+  await driver.saveScreenshot('./flutter-tab.png');
+
+  assert.strictEqual(await driver.getElementText(counterTextFinder), '2');
+
+  driver.deleteSession();
+})();
```
Comment (nghiadhd) [inline flutter/javascript/src/index.js]: Add newline

File: flutter/javascript/src/index.js (hunk 3 above)
Comment (hyndole) [reply flutter/javascript/src/index.js:33]: I fixed. Thanks!

File: flutter/javascript/package.json (hunk 2 above)
Comment (hyndole) [reply flutter/javascript/package.json:2]: I fixed. Thanks!

File: flutter/javascript/.babelrc (hunk 1 above)
Comment (hyndole) [reply flutter/javascript/.babelrc]: I fixed. Thanks!

File: flutter/javascript/src/index.js (hunk 4 above)
Comment (hyndole) [reply flutter/javascript/src/index.js]: I fixed. Thanks!

File: flutter/javascript/.babelrc [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+
```
Comment (thanhtan27081998) [inline flutter/javascript/.babelrc]:
```suggestion
  
  
```

Comment (thanhtan27081998) [reply flutter/javascript/.babelrc]: Add new line

**Review by thanhtan27081998 — COMMENTED** [review]
LGTM

**Pull request merged**